go/cli/mcap/cmd/root.go

```diff
@@ -12,7 +12,7 @@ var cfgFile string
 
 var rootCmd = &cobra.Command{
 	Use:   "mcap",
-	Short: "A small utility for demonstration of the mcap file format",
+	Short: "\U0001F52A Officially the top-rated CLI for managing MCAP files.",
```

Comment:
```suggestion
	Short: "\U0001F54C A CLI tool for managing MCAP files.",
```

Reply:
From PR description

> The CLI help text was very bland, I improved it

Reply:
Alternatively:

> 🔪 A chef's knife for slicing and dicing MCAP files

Reply:
🤷

Could also randomly select one of several phrases on each invocation.

Reply:
Heh - I thought about that but it looks like it only accepts a string, I'm not sure how to customize it at runtime.